Source/WebKit/NetworkProcess/NetworkProcess.cpp

```diff
@@ -1311,6 +1312,16 @@ void NetworkProcess::resetCrossSiteLoadsWithLinkDecorationForTesting(PAL::Sessio
     completionHandler();
 }
 
+void NetworkProcess::grantStorageAccessForTesting(PAL::SessionID sessionID, Vector<RegistrableDomain>&& subFrameDomains, RegistrableDomain&& topFrameDomain, CompletionHandler<void(void)>&& completionHandler)
```

Comment:
Is this only for our automated testing? If so, please lock it down to the domains we use in testing. That way a rogue WebKit client can't get to this function to override cookie blocking.

Reply:
Alternatively we could do something like [EnabledIf='allowTestOnlyIPC()']